### DISCUSSION TITLE
How to correctly apply drop_valuation to quarterly triangles?

### DISCUSSION THREAD

**Discussion by contenrico** [casact/chainladder-python#455] [General]
Hello,

I'm working with a quarterly triangle and I'm trying to exclude the latest diagonal (valuation date 2022-12). This is the code I'm running:

`cl.Development(drop_valuation='2022').fit_transform(incurred).link_ratio`

However, the excluded link ratios are the following:

![image](https://github.com/casact/chainladder-python/assets/137546866/681a2528-fda7-485e-bd3e-8b1000bea142)

This is quite odd, as I would expect the following points to be excluded:
(2022, 3), (2022, 6), (2022, 9)
(2021, 15), (2021, 18), (2021, 21)
(2020, 27), (2020, 30), (2020, 33)
etc.

Is there a way to use drop_valuation that achieves this?

Thanks!

**Comment by jbogaardt** [comment]
That is odd, and not the desired behavior.  According to the docs, `drop_valuation`  supports a list of valuations. Have you tried passing in `drop_valutation=[('2022-03, '2022-06', '2022-09')]`?

**Reply by contenrico** [reply]
I think that specific syntax is not supported as I get the following error:

`ValueError: Buffer has wrong number of dimensions (expected 1, got 2)`

However I've tried passing this in:

`cl.Development(drop_valuation=['2022-03', '2022-06', '2022-09']).fit_transform(incurred).link_ratio`

And I still get the same result unfortunately. If it helps, this is the configuration of my triangle object:

![image](https://github.com/casact/chainladder-python/assets/137546866/fabdd0a8-8f9a-4d00-994f-3ba142ab6dc9)

**Reply by contenrico** [reply]
Hi @jbogaardt,

Any ideas regarding the above? Tried a few different approaches but none of them seem to work

**Reply by jbogaardt** [reply]
Hello @contenrico, sorry, it's been a busy week and haven't been able to thoroughly debug or provide an alternate path. I think this question should ultimately be updated to a bug report. As this is existing functionally that should work, we would slate it in for next release.

**Reply by contenrico** [reply]
No worries - and thank you! I'll create an issue from this discussion then
